### DISCUSSION TITLE
Whether the TAB supports nesting

### DISCUSSION THREAD

**Discussion by Hello-dl** [carbon-design-system/carbon#11737] [Help]
I'm using Tabs/TabList/TabPanels/TabPanel to created 3 tabs in one page: Tab1, Tab2 Tab3;  then 6 more tabs are added under Tab2: Tab2-1, Tab2-2, Tab2-3, Tab2-4, Tab2-5,  Tab2-6,   however, when I click Tab2-1, it shows content of tab1 instead of Tab2-1; When I click tab2-3, TAB3 is displayed... So I wonder if tabs don't support nesting？

<img width="645" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/57393224/177235717-804c70ff-8943-48eb-b88a-c317c3dce479.png">

<img width="541" alt="Screen Shot 2" src="https://user-images.githubusercontent.com/57393224/177235729-35867c8b-97d4-444f-9cb0-83aa5125ddef.png">

<img width="983" alt="Screen Shot 2022-07-05 at 10 09 31" src="https://user-images.githubusercontent.com/57393224/177235735-9f0ff4db-2ccb-43c2-886a-6a8f9893e424.png">

**Comment by tay1orjones** [comment]
Hi! Could you reproduce the issue in a [codesandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/v11.8.0/examples/codesandbox)? It’s hard to know exactly what might be going wrong without seeing the issue first-hand.
